include/diaspora.php

```diff
@@ -880,11 +880,14 @@ private static function contact_by_handle($uid, $handle) {
 		if (dbm::is_result($r)) {
 			return $r[0];
 		} else {
-			// We haven't found it?
-			// We use another function for it that will possibly create a contact entry
+			/*
+			 * Not found yet? Then use another function for it that will
+			 * possibly create a contact entry.
+			 */
 			$cid = get_contact($handle, $uid);
 
 			if ($cid > 0) {
+				/// @TODO Contact-retrival should be encapsulated into an "entity" class like `Contact`
```

Comment:
guess it is more about the spelling. `retrieval` is the correct spelling not `retrival` ;-)